code/FulpstationCode/sec_borg_files/fulp_sec_borg.dm

```diff
@@ -0,0 +1,477 @@
+//Pepperspray Module
+
+/obj/item/reagent_containers/spray/pepper/cyborg

```

Comment:
could you give these to PK borgs too? it fits their MO

Reply:
In another PR maybe.